oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/oidc/authentication/OidcIdTokenValidatorTests.java

```diff
@@ -46,9 +47,11 @@
 	private Instant issuedAt = Instant.now();
 	private Instant expiresAt = this.issuedAt.plusSeconds(3600);
 	private Duration clockSkew = Duration.ofSeconds(60);
+	private OidcIdTokenValidator idcIdTokenValidator;
 
 	@Before
 	public void setup() {
+		this.idcIdTokenValidator = new OidcIdTokenValidator(this.registration.build());
```

Comment:
Now that we have the member `idTokenValidator`, let's ensure `validateIdToken()` is using it.